server/routes/events.js

```diff
@@ -32,5 +33,44 @@ router.get('/user/events/', isAuthenticated, async (req, res) => {
     }
 })
 
+//update user's status for an event
+router.patch('/user/events/:event_id', async (req, res) => {
+    const event_id = parseInt(req.params.event_id);
+    const {updatedStatus} = req.body; //wasn't sure if user_id should be a path parameter because it may be sensitive information
```

Comment:
Do you need the user_id as a path parameter? It seems this data is derivable from the event id. You could also pass it in the body as well under a param like actorID